bindings/matlab/+iDynTreeWrappers/README.md

```diff
@@ -73,16 +73,20 @@ The purpose of the wrapper is therefore to provide a simpler and easy-to-use int
 
 Not proper wrappers, they wrap more than one method of the class each. **Requirements:** compile iDyntree with Irrlicht `(IDYNTREE_USES_IRRLICHT = ON)`.
 
+**Warning**: the visualizer class may be deprecated in a future release.
```

Comment:
According to the documentation of this functions, they are being deprecated, so I would avoid the `may be`. However, as far as I remember back in time with @fjandrad we decided not to deprecate this functions as th new visualizer was not working in Octave.